packages/services/src/kernel/future.ts

```diff
@@ -17,19 +17,30 @@ import {
   KernelMessage
 } from './messages';
 
+// from Phosphor
+export
```

Comment:
Does this need to be exported?

Reply:
no. Fixed in the latest commit I just pushed.